tests/storage/iso/test_iso_sr_smb_reboot.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from lib.common import wait_for
+from conftest import copy_tools_iso_to_iso_sr, iso_mount_and_read_test_from_vm, clean_iso_sr
+
+@pytest.mark.usefixtures("smb_iso_sr")
+class TestISOSRsmb():
+    """
+    Test that use fixtures to setup the SR and import VMs.
+    This test is longer because the host reboot between creation of the sr and the test with the iso file.
+    We Copy the tools iso on the sr, then mount it inside the VM to validate we can access it.
```

Comment:
Same comments as in other reboot tests.

Reply:
i don't understand this comment. Can i have some details please ?

Reply:
First line to remove, second line to fix, last line to remove or fix.